src/UserService.Business/Commands/User/CreateUserCommand.cs

```diff
@@ -235,19 +239,18 @@ public OperationResultResponse<Guid> Execute(CreateUserRequest request)
 
             var dbUser = _mapperUser.Map(request, avatarImageId);
 
-            Guid userId = _userRepository.Create(dbUser, request.Password);
+            var password = request.Password != null ? request.Password : _generatePassword.Execute();
```

Comment:
You  can do it like var password = request.Password ?? _generatePassword.Execute();

Reply:
Done